badger_api/requests.py

```diff
@@ -64,3 +65,13 @@ def fetch_all_claimable_balances(chain: str):
             data = future.result()["rewards"]
             results = {**results, **data}
     return results
+
+
+@lru_cache
+def fetch_token_names(chain: str):
+    return requests.get(f"{badger_api}/tokens?chain={chain}").json()
+
+
+def fetch_token(chain: str, token: str):
+    token_names = fetch_token_names(chain)
```

Comment:
try catch to handle error responses

Reply:
We can address this for all the requests in a separate PR